src/style.css

```diff
@@ -1 +1,69 @@
 /* Styles go here */
+:root {
+  --colorForlinksHover: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
```

Comment:
`box-sizing` should be in * selector.

Reply:
Check the previous comments. if you have any questions just feel free to ask in fe_chat